Section_III/software-challenge.tex

```diff
@@ -0,0 +1,42 @@
+\clearpage
+\sffamily
+{\bfseries\color[rgb]{0.4,0.4,0.4} Part F: Software Challenge}
+\phantomsection
+\addcontentsline{toc}{subsection}{Part F: Software Challenge}
+
+\bigskip
+\added{
+In addition to the technical challenges, a software challenge is conducted to allow teams to present their recent research.
+
+\smallskip
+
+Participating teams have to create a poster (A3 or A2), and prepare a short 5 minutes oral presentation which additionally explains and shows the idea and results of this approach.
+If there is a monitor on site, it can be used for the presentation. 
+However, this depends on the location and the teams should be prepared to give their presentation only with their poster if necessary.
+
+\smallskip
+
+Teams are encouraged to publish the code for their approach to enable a fast progress in the league. 
+However, only the top 3 ranked teams are required to publish their code
+with instructions within one month after the RoboCup 2022.
+\bigskip
+
+{\bfseries Scoring}
+\smallskip
+
+The winner will be decided by a vote among the team leaders using the Borda count mechanism\footnote{\url{http://en.wikipedia.org/wiki/Borda_count}}. 
+Each participating team will vote for their top 5 teams in order (excluding themselves). 
+Teams are encouraged to evaluate the presentations based on the following criteria:
+
+\begin{itemize}
+	\item Novelty
+	\item Benefit to the league
+	\item Documentation
+	\item Ease of transfer to other teams
+\end{itemize}
+
+At a time decided by the designated referee, within one hour of the last demonstration if not otherwise specified, the captain of each team will submit the team's rankings by filling out an online form. 
```

Comment:
I would remove "online" here to allow us to use a printed form if we do not have a stable internet connection at the venue